src-tauri/src/mesh/device/mod.rs

```diff
@@ -143,7 +143,8 @@ pub struct MeshDevice {
     pub config: protobufs::LocalConfig, // local-only device configuration
     pub my_node_info: protobufs::MyNodeInfo, // debug information specific to device
     pub nodes: HashMap<u32, MeshNode>, // network devices this device has communicated with
-    pub region_unset: bool,       // flag for whether device has an unset LoRa region
+    pub edges: HashMap<(u32, u32), (f64, u64)>, // edges between nodes (node_id, neighbor_id) -> (snr, timestamp)
```

Comment:
Storing edges as a hashmap in the `MeshDevice` struct. While eventually this struct may contain raw `NeighborInfo` packets, they don't exist as of now, so we'll just store raw ids, snrs, and timestamps.